### PR TITLE
{bio}[foss/2020b] metaWRAP v1.3 (+ deps) w/ Python 2.7.18

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#12726~~ (bmtagger), ~~#12720~~ (CheckM), ~~#12724~~ (MEGAHIT), ~~#12725~~ (CONCOCT), ~~#12727~~ (MaxBin), ~~#12728~~ (QUAST), ~~#12729~~ (Trim_Galore), ~~#12730~~ (Seaborn), ~~#12732~~ (taxator-tk)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (6 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/9288fcc6e11589ad1aed0504f036463f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/22a00985103cc860e3f58318d2c669b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3163.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/9728e878c16aa44a0b25427179686ebc for a full test report.

File: easybuild/easyconfigs/k/KronaTools/KronaTools-2.8-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contribution from the Crick HPC team
```
Comment (branfosj) [inline easybuild/easyconfigs/k/KronaTools/KronaTools-2.8-GCCcore-10.2.0.eb:1]: See  #12871 for suggested changes to the existing KronaTools ecs. This should be changed in the same way.

File: easybuild/easyconfigs/k/KronaTools/KronaTools-2.8-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+several Bioinformatics tools as well as from text and XML files."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/marbl/Krona/releases/download/v%(version)s/']
+sources = [{
+    'filename': '%(name)s-%(version)s.tar',
+    'extract_cmd': 'tar xvf %s -C %(installdir)s/../',
+}]
+
```
Comment (branfosj) [inline easybuild/easyconfigs/k/KronaTools/KronaTools-2.8-GCCcore-10.2.0.eb:24]: checksum missing

File: easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.15-gompi-2020b-Python-2.7.18.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+version = '2.15'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bitbucket.org/berkeleylab/metabat'
+description = "An efficient tool for accurately reconstructing single genomes from complex microbial communities"
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+source_urls = ['https://bitbucket.org/berkeleylab/metabat/get/']
+sources = ['v%(version)s.tar.gz']
```
Comment (branfosj) [inline easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.15-gompi-2020b-Python-2.7.18.eb:13]: checksum missing

File: easybuild/easyconfigs/m/metaWRAP/metaWRAP-1.3-foss-2020b-Python-2.7.18.eb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+homepage = 'https://github.com/bxlab/metaWRAP'
+description = """MetaWRAP aims to be an easy-to-use metagenomic wrapper suite
+that accomplishes the core tasks of metagenomic analysis from start to finish:
+read quality control, assembly, visualization, taxonomic profiling, extracting
+draft genomes (binning), and functional annotation."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/bxlab/metaWRAP/archive/']
+sources = ['v%(version)s.tar.gz']
```
Comment (branfosj) [inline easybuild/easyconfigs/m/metaWRAP/metaWRAP-1.3-foss-2020b-Python-2.7.18.eb:16]: checksum missing

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node2618.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/38af82e07736b4efebc61028134903e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3506.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/1a88ba512374a7e13bf31aa3b7cf153f for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a63837d437d3f3a41b27c414311c0ade for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**